app/controllers/application_controller.rb

```diff
@@ -1,8 +1,11 @@
 # frozen_string_literal: true
 
 require "jwt"
+require "auth0_client"
 
 class ApplicationController < ActionController::API
+  include Secured
```

Comment:
I know nothing about Ruby but is this all you have to do to enable SSL?
Don't we need to add a certificate?  As well

Reply:
No, this line mixes in the `Secured` module that he defined in another file in this PR, at app/controllers/concerns/secured.rb. This PR doesn't do anything related to SSL; we terminate SSL at the Kubernetes ingress level, so by the time the request hits the Rails server, it's already speaking HTTP, not HTTPS.

Reply:
ah ok thanks